apps/ledger-live-desktop/src/renderer/modals/AddAccounts/steps/StepChooseCurrency.tsx

```diff
@@ -258,7 +277,12 @@ export const StepChooseCurrencyFooter = ({
         <Button
           primary
           disabled={!currency || fullNodeNotReady || !navigator.onLine}
-          onClick={() => transitionTo("connectDevice")}
+          onClick={() => {
+            if (mock) {
```

Comment:
remove this piece of code according to the discussion https://ledger.slack.com/archives/C0308JT8WS2/p1692373876296589?thread_ts=1691507827.835819&cid=C0308JT8WS2